bootstrap/init.go

```diff
@@ -0,0 +1,78 @@
+package bootstrap
+
+import (
+	"fmt"
+	"os"
+	"text/template"
+
+	"gofr.dev/pkg/gofr"
+)
+
+const (
+	modContent = `module {{ .Module }}
+
+go 1.22.4
+
+require gofr.dev v{{ .GofrVersion }}
+`
+	mainContent = `package main
+
+import (
+	"gofr.dev/pkg/gofr"
+)
+
+func main() {
+	app := gofr.New()
+
+	app.GET("/hello", func(ctx *gofr.Context) (interface{}, error) {
+		return "Hello World!", nil
+	})
+
+	app.Run()
+}
+`
+	fileMode = 0644
+)
+
+type modInfo struct {
+	Module      string
+	GofrVersion string
+}
+
+func Create(ctx *gofr.Context) (interface{}, error) {
+	name := ctx.Param("name")
+	gofrVersion := ctx.Param("gofr")
+
+	if gofrVersion == "" {
+		gofrVersion = "1.15.0"
```

Comment:
Can we use the latest version of GoFr here i.e. `v1.17.0` ?

Reply:
This defaults to v1.15.0, and we cannot always keep it on par with the latest one, but sure I can change this as long as this PR is in progress.

Reply:
Sure! Thankyou